src/models/forms.js

```diff
@@ -3,13 +3,31 @@
 import neuburgForm from './forms/neuburgForm'
 import mongoose from 'mongoose'
 import cityConfigs from '../cities/cityConfigs'
+import _ from 'lodash'
+import Offer from './Offer'
+
+type AllFormsType = | typeof neuburgForm;
 
 const schemaOptions = {strict: 'throw'}
 
 const Neuburg = mongoose.model(
   cityConfigs.neuburgschrobenhausenwohnraum.cmsName, mongoose.Schema(neuburgForm, schemaOptions)
 )
 
+const addNotIncludedFor = (offer: Offer, form: AllFormsType, names: Array<string>, omit: Array<string>) => {
+  names.forEach((name: string) => {
+    const diff = _.difference(_.get(form, name)['enum'], _.get(offer, `formData.${name}`))
+    _.pull(diff, ...omit)
+    _.set(offer, `formData.${name}Diff`, diff)
+  })
+}
+
 export default {
-  [cityConfigs.neuburgschrobenhausenwohnraum.cmsName]: Neuburg
+  [cityConfigs.neuburgschrobenhausenwohnraum.cmsName]: {
+    Schema: Neuburg,
```

Comment:
Is it really a schema? `Neuburg` was assigned a `mongoose.model(...)` call. Therefore I'd call it a `Model` - maybe even `FormModel`  - rather than a `Schema`. Not sure though if that's right in the mongoose context.

Reply:
You are right.